src/__test__/ApplePay.test.ts

```diff
@@ -41,55 +46,58 @@ jest.mock('react-native', () => {
   return mockReactNative;
 });
 
+const nativeEventEmitter = new EventEmitter();
+
 describe('Test Apple Pay', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  it('initializeApplePay works as expected', async (done) => {
+  it('initializeApplePay works as expected', async () => {
     expect.assertions(2);
     try {
       const spyVerifyStringType = jest.spyOn(Utilities, 'verifyStringType');
       const testApplePayMerchantId = 'testMerchantId';
       await SQIPApplePay.initializeApplePay(testApplePayMerchantId);
       expect(spyVerifyStringType).toHaveBeenCalledTimes(1);
       expect(NativeModules.RNSQIPApplePay.initializeApplePay).toHaveBeenCalledTimes(1);
-      done();
```

Comment:
Why do we remove `done()`?

Reply:
It is not needed and becomes obsolete in the typescript test case.